nailgun/entities.py

```diff
@@ -949,7 +951,12 @@ def logs(self, synchronous=True, **kwargs):
 
 
 class DockerHubContainer(AbstractDockerContainer):
-    """A docker container that comes from Docker Hub."""
+    """A docker container that comes from Docker Hub.
+    .. WARNING:: 'repository_name' field here refer to the name of the
+    image repository from your docker hub resource and doesn't have any
+    reference to repositories created in the application itself
```

Comment:
The `WARNING` directive acts like the `param` and `returns` directive. It should be separated from surrounding text and successive lines indented, or else Sphinx doesn't parse it correctly:

![screenshot_2015-08-17_09-37-53](https://cloud.githubusercontent.com/assets/1012989/9306020/a6a82b64-44c3-11e5-81f1-b83963b757a8.png)

I've fiddled with the wording below in an attempt to make it flow more nicely.

```
    """A docker container that comes from Docker Hub.

    .. WARNING:: The ``repository_name`` field references an image repository
        on the `Docker Hub <https://hub.docker.com/>`_, not a locally created
        :class:`nailgun.entities.Repository`.

    """
```